public/components/main/loading_modal/loading_modal.tsx

```diff
@@ -14,7 +14,7 @@ import {
   EuiFlexGroup,
   EuiFlexItem,
   EuiLoadingSpinner,
-  EuiButton,
```

Comment:
Should this just be removed, seems not used?

Reply:
yep; not used.